game/menu.cc

```diff
@@ -39,7 +44,7 @@ void Menu::move(int dir) {
 	else if (dir < 0 && selection_stack.back() > 0) --selection_stack.back();
 }
 
-void Menu::select(unsigned sel) {
+void Menu::select(size_t sel) {
```

Comment:
why a size_t? (uint32_t) sel is the "current selection" it doesn't represent a size of some sort.

Reply:
Because it is compared against a size()